### PR TITLE
Updating geth-sdk with changes from generic-evm-sdk

### PR DESCRIPTION
**Pull request by ash-krishnan**
Fixes # .

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -814,7 +814,8 @@ func (ec *SDKClient) GetGasPrice(
 ) (*big.Int, error) {
 	var gasPrice *big.Int
 	var err error
-	if input.GasPrice == nil {
+	if input.GasPrice == nil || input.GasPrice.Uint64() == 0 {
+		log.Println("Fetching gas price")
```
Comment (shrimalmadhur) [inline client/client.go:818]: these logs needed or they are just for debugging?

File: examples/ethereum/config/config.go [hunk 2]
```diff
@@ -249,15 +249,15 @@ func LoadConfiguration() (*configuration.Configuration, error) {
...
+	// payload := []configuration.Token{}
 	config.RosettaCfg = configuration.RosettaConfig{
 		SupportRewardTx: true,
 		TraceType:       configuration.GethNativeTrace,
 		Currency: &RosettaTypes.Currency{
 			Symbol:   "ETH",
 			Decimals: 18,
 		},
-		TokenWhiteList: payload,
+		// TokenWhiteList: payload,
```
Comment (xiaying-peng) [inline examples/ethereum/config/config.go:260]: we need whitelist feature, otherwise rosetta-ethereum wont work

**Comment by xiaying-peng** [comment]
@ash-krishnan Can you update this PR by removing whitelist changes

File: client/client.go (hunk 1 above)
Comment (ash-krishnan) [reply client/client.go:818]: not just for debugging, added more logs for clarifying purposes

File: examples/ethereum/config/config.go (hunk 2 above)
Comment (ash-krishnan) [reply examples/ethereum/config/config.go:260]: ah i'll uncomment this out

**Pull request closed without merging**